client/src/components/common/Sidebar/SidebarItem/SidebarItem.js

```diff
@@ -1,27 +1,47 @@
-import { Link } from 'react-router-dom';
-import { InputGroup, FormControl } from 'react-bootstrap';
+import { useState, useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import '../../../../App.css';
 
 const SidebarItem = ({ ...props }) => {
-  const { product } = props;
-  const { title, description, quantity, unit } = product;
+  const { product, basketProducts, setBasketProducts } = props;
+  const { name, description, quantity, unit, price } = product;
+  const [totUnitPrice, setTotUnitPrice] = useState(price * quantity);
+
+  const handleRemoveItem = (pid, fid) => {
+    let newBasket = [...basketProducts];
+    console.log(pid + ' ' + fid);
+    newBasket = newBasket.filter(
+      item => !(item.pid === pid && item.fid === fid)
+    );
+    console.log(newBasket);
+    setBasketProducts(newBasket);
+  };
+
+  useEffect(() => {
+    setTotUnitPrice(price * quantity);
+  }, [basketProducts]);
+
   return (
```

Comment:
Maybe here could be useful to add a fast check on  ` quantity ` sign. If it is  ` -1 `  return some error.